Development/src/pages/subscriptions/SubscriptionsCreate.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+    BooleanInput,
+    Create,
+    ListButton,
+    NumberInput,
+    SimpleForm,
+    TextInput,
+    TopToolbar,
+} from 'react-admin';
+import ObjectInput from '../../components/ObjectInput';
+import QueryVersion from '../../components/QueryVersion';
+import RawButton from '../../components/RawButton';
+
+const SubscriptionsCreateActions = ({ basePath, data, resource }) => (
+    <TopToolbar>
+        {data ? <RawButton record={data} resource={resource} /> : null}
+        <ListButton basePath={basePath} />
+    </TopToolbar>
+);
+
+const SubscriptionsCreate = props => (
+    <Create actions={<SubscriptionsCreateActions />} {...props}>
+        <SimpleForm redirect="show">
+            <TextInput source="resource_path" label="Resource Path" />
```

Comment:
Can we add a set of allowed values for `resource_path`? Maybe using [SelectInput](https://marmelab.com/react-admin/Inputs.html#selectinput)